lib/render.js

```diff
@@ -80,7 +80,7 @@ export function renderRoute (url, context = {}) {
     if (!context.nuxt.serverRendered) {
       app = '<div id="__nuxt"></div>'
     }
-    const publicPath = self.options.build.publicPath.indexOf('http') === 0 ? self.options.build.publicPath : urlJoin(self.options.router.base, self.options.build.publicPath)
+    const publicPath = self.options.build.publicPath
```

Comment:
Why removing the ternary? If the public path is `/app/` and `router.base` is `/my-site/`, publicPath should be `/app/my-site/`